generator.rb

```diff
@@ -46,13 +26,15 @@
         else
           puts "Unique Locator"
           #TODO: A method should be created to replace characters
-          out_file.puts("  " + node['name'].upcase.tr(" ", "_").tr("-", "_").tr(":", "") + " = { name: '" + node['name'] + "' }")
```

Comment:
Should have a method that takes a string and returns it with all these characters replaced.

Reply:
tr(" ", "_").tr("-", "_").tr(":", "")